src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEFieldSymbol.cs

```diff
@@ -212,6 +212,8 @@ private void EnsureSignatureIsLoaded()
                 TypeSymbol type = (new MetadataDecoder(moduleSymbol, _containingType)).DecodeFieldSignature(_handle, out isVolatile, out customModifiers);
                 ImmutableArray<CustomModifier> customModifiersArray = CSharpCustomModifier.Convert(customModifiers);
                 type = DynamicTypeDecoder.TransformType(type, customModifiersArray.Length, _handle, moduleSymbol);
+
+                type = TupleTypeSymbol.TransformToTupleIfCompatible(type); // temporary shallow unification
```

Comment:
You guys are tracking this work already. This is where the tuple transform will plug in.

---
In reply to: [62216797](https://github.com/dotnet/roslyn/pull/11047#discussion_r62216797) [](ancestors = 62216797)